### ISSUE TITLE
Can't query for fields on a type

### ISSUE THREAD

**Issue by steveluscher** [graphql/graphiql#23]
This query results in `type.getFields is not a function`.

```
query ShipTypeQuery {
  __type(name:"Starship") {
    fields
  }
}
```

*Edit*: s/Ship/Starship/

**Comment by leebyron** [comment]
Client side or server side? I haven't been able to repro this yet

**Comment by leebyron** [comment]
Is "Ship" a type in your schema? I may need to see your machine to repro

**Comment by fson** [comment]
I was able to reproduce this on http://graphql-swapi.parseapp.com/graphiql/

Seems that the combination of nonexistent type "Ship" and missing selection set for `fields` might trigger it somehow.

**Comment by steveluscher** [comment]
Sorry for all of the derp in this bug report. Let me try again.

1. Visit http://graphql-swapi.parseapp.com/graphiql/
2. Type, into the query, `allStarships` and press ⌘-Enter; fields autocomplete.
3. Empty out the subfields, type `edges`, and press ⌘-Enter; `type.getFields is not a function` error appears.

![getfields](https://cloud.githubusercontent.com/assets/13243/9590078/500e9ea8-4fe7-11e5-8a6b-301605464b33.gif)

**Comment by leebyron** [comment]
Thanks for the steps! I'll dig into this.

BTW, what do you use to make screencast gifs?

**Comment by wincent** [comment]
@leebyron: [This](https://gist.github.com/dergachev/4627207) is a pretty nice resource. I've used the [https://github.com/dergachev/screengif](screengif) tool mentioned there with success before.

**Comment by steveluscher** [comment]
I use [Licecap](http://www.cockos.com/licecap/).

**Comment by leebyron** [comment]
On npm now as v0.1.3
